apicompat/netstandard1x/netstandard1x.depproj

```diff
@@ -1,14 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
-    <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)'==''">.NETStandard,Version=v1.6</NuGetTargetMoniker>
     <TargetFramework Condition="'$(TargetFramework)'==''">netstandard1.6</TargetFramework>
```

Comment:
Do we really need a default value here and if yes should we update it to netstandard2.0 or 2.1?

Reply:
Right now we just build this for netstandard1.0-1.6 & net47. I'll chat with @terrajobst later today about if we should baseline for 2.0/2.1 as well.

Reply:
https://github.com/dotnet/standard/issues/1029